reference/as_architecture.adoc

```diff
@@ -22,9 +22,9 @@ used to connect/federate/transform datasources to produce a `.vdb` file.
 * **quickstarts** - A maven quickstart showing how to utilize {{ book.productnameFull }}.
 {% endif %}
 
-{% if book.targetWildfly %}
-image:images/teiid-architecture.png[images/teiid-architecture.png]
-{% endif %}
+
+image:images/data-virt-arch.png[images/data-virt-arch.png]
+
```

Comment:
The new image isn't very applicable to wildfly - especially the operator.  While it is possible to containerize Teiid with Wildfly, we aren't emphasizing that at all.  We can conditionalize the new to only spring boot.

Reply:
Should we retain the old version of the diagram (teiid-architecture.png) for Wildfly? There's also a 
![data_virt_engine](https://user-images.githubusercontent.com/23705736/75887394-bb5d9880-5df7-11ea-846a-ca776c4dc3dc.png) diagram yet to be inserted into the Architecture topic. Should that likewise be conditionalized to SB only?

Reply:
I believe old one has few more boxes in it, So I leave as is for Wildfly with old diagram

Reply:
The consensus is to leave the old diagram for Wildfly.  

The DV engine image is not specific to spring boot, but probably calls netty out too prominently.  There are effectively 2 client transports - our netty sockets and the web endpoints (which can be odata or rest).

Reply:
ok, let me close this pr and resubmit without deleting the old version. I'll also implement the conditionalization as discussed, and add the new DV Engine diagram.
@rareddy is there any specific context that I should provide in introducing the DV engine diagram, or should I essentially say, _hey, here's a more detailed view of the internals of the DV service_?

Reply:
IMO, no additional context is needed